.github/workflows/consuming.yml

```diff
@@ -41,36 +41,39 @@ jobs:
           - project: submariner-operator
             k8s_version: '1.24'
     steps:
-      - name: Check out the Shipyard repository
-        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
-
-      - name: Build the latest Shipyard image
-        run: make images
-
       - name: Check out the ${{ matrix.project }} repository
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
         with:
           # This is replaced to stable branch by auto release process
           ref: devel
           repository: submariner-io/${{ matrix.project }}
-          path: ${{ matrix.project }}
+
+      - name: Check out the Shipyard repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          path: shipyard
+
+      - name: Build the latest Shipyard image
+        run: make -C shipyard images
 
       - name: Copy Shipyard resources
-        run: cp -n Dockerfile.* Makefile.dapper .dapper ${{ matrix.project }}/
+        run: cp -n shipyard/{Dockerfile.*,Makefile.dapper,.dapper} .
 
       - name: Make sure ${{ matrix.project }} is using the built Shipyard image
-        run: sed -i 's/shipyard-dapper-base:*.*/shipyard-dapper-base:dev/' ${{ matrix.project }}/Dockerfile.dapper
+        run: sed -i 's/shipyard-dapper-base:*.*/shipyard-dapper-base:dev/' Dockerfile.dapper
+
+      - name: Make sure ${{ matrix.project }} is using the Shipyard go module
+        run: make go-replace REPLACE=github.com/submariner-io/shipyard=./shipyard
```

Comment:
This is the significant change — most of the rest of this PR is the result of inverting the extraction. Is there any particular reason to do so? The commit message doesn’t explain that part of the change.

Reply:
Not sure what you mean?

What's missing from the commit message? It already specifies a reason:
```
Consuming projects running E2E tests rely on code supplied by Shipyard.
Use the local Shipyard when testing them, to make sure the latest code
is tested, and catch any breakages.
```

Reply:
The commit message explains these two lines. The PR also inverts the extraction: previously we checked out Shipyard, then checked out the project undergoing testing into a subdirectory; but the PR changes that so that the project being tested is checked out, and then Shipyard is checked out into a subdirectory of that. The commit message doesn’t explain that change. Is it actually necessary?